app/gateway/changelog.md

```diff
@@ -753,6 +753,50 @@ This should be a 400 because the configuration is invalid.
 * When the OpenID Connect (OIDC) plugin is configured to reference HashiCorp Vault in the `config.client_secret` field (for example, `{vault://hcv/clientSecret}`),
 it does not look up the secret correctly.
 
+## 3.2.2.4
+**Release Date** 2023/08/21
+
+### Enterprise
+
+#### Breaking Changes
+
+- Ubuntu 18.04 artifacts are no longer supported as they are end-of-life.
+- Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+
+#### Deprecations
+
+- **CentOS packages are now removed from the release and are no longer supported in future versions.**
+
+#### Dependencies
+
+* `lua-resty-kafka` is bumped from 0.15 to 0.16
+
+#### Fixes
+
+* Updated the datafile library to make the SAML plugin work again when Kong is controlled by systemd.
+* Fixed an issue where the anonymous report couldn't be silenced by setting `anonymous_reports=false`.
+* Fixed an issue where a crashing Go plugin server process would cause subsequent requests proxied through Kong to execute Go plugins with inconsistent configurations. The issue only affected scenarios where the same Go plugin is applied to different route or service entities. [#6208](https://github.com/Kong/kong-ee/pull/6208) THIS IS STILL IN PROGRESS?
```

Comment:
This one is still in progress in Jira and GH, so just noting that in case it doesn't make it in.

Reply:
You'll want to remove the link to kong-ee before this goes in.